python-avd/pyavd/_eos_designs/structured_config/network_services/utils.py

```diff
@@ -66,6 +66,10 @@ def _vrf_default_ipv4_subnets(self: AvdStructuredConfigNetworkServicesProtocol)
                 if subnet not in subnets:
                     subnets.append(subnet)
 
+                for ip_address_secondary in svi.ip_address_secondaries:
+                    subnet = str(ipaddress.ip_network(ip_address_secondary, strict=False))
+                    subnets.append(subnet)
```

Comment:
We should probably check if subnet is already there otherwise we may have same prefix duplicated in prefix list

```suggestion
                for ip_address_secondary in svi.ip_address_secondaries:
                    subnet = str(ipaddress.ip_network(ip_address_secondary, strict=False))
                    if subnet not in subnets:
                        subnets.append(subnet)

```

Reply:
Added condition